### PR TITLE
fix #81146 www.nu.nl

### LINKED ISSUE #81146: www.nu.nl
### Issue URL (Ads)
 [https://www.nu.nl/achterklap/6129121/john-travolta-vond-het-lastig-om-anderen-te-troosten-na-overlijden-vrouw.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.nu.nl%2Fachterklap%2F6129121%2Fjohn-travolta-vond-het-lastig-om-anderen-te-troosten-na-overlijden-vrouw.html)
### Comment
Username: @pauperx
> Dear support,
> 
> Please see the attached screenshot. I want to block the highlighted ad in the screenshot ('NUshot'). That works for one day by using the Element Blocking tool from AdGuard. The next day the number of the ad has changed, and I have to block the ad again. Is there a possibility to use a wildcard or something to have these ads blocked permanently instead of just for one day?
> 
> Kind regards!

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> EasyList Czech and Slovak, <br/>EasyList Italy, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
Referrer: | [https://www.nu.nl/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.nu.nl%2F)

### PR DESCRIPTION
**Pull request by stanislav-atr**
#81146

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -141,6 +141,8 @@ neostencil.com###enquirefooter
 neostencil.com###fixedheaderform238
 neostencil.com###demo-snackbar-example
 neostencil.com###hubSpotFormDialog
+nu.nl#$#.webpush-popup { display: none !important; }
+nu.nl#$#body { overflow: visible !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:145]: `nu.nl#$?#body { overflow: visible !important; }`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by BlazDT** [comment]
@zloyden Why #$?# ? It is not an ext-CSS.

**Comment by zloyden** [comment]
> @zloyden Why #$?# ? It is not an ext-CSS.

This is required for apps. We need to use this to get around their style with `!important`.

**Comment by AdamWr** [comment]
I think that using more specific path like `html > body[class]` or `body:not(#oh_no_important)` should has higher priority and also should works fine. Just saying/informing.

**Pull request merged**